configure

```diff
@@ -1599,11 +1672,11 @@ fi
 if [ -n "$BSON_PATH" ]; then
     XLIBS="$XLIBS -lbson-1.0"
 fi
-if [ -n "$FREERDP2_PATH" ]; then
-    XLIBS="$XLIBS -lfreerdp2"
+if [ -n "$FREERDP3_PATH" ]; then
+    XLIBS="$XLIBS -lfreerdp3"
 fi
-if [ -n "$WINPR2_PATH" ]; then
-    XLIBS="$XLIBS -lwinpr2"
+if [ -n "$WINPR3_PATH" ]; then
+    XLIBS="$XLIBS -lwinpr3"
```

Comment:
here you have not reestablished the previous librdp2 for XLIBS, only librdp3

Reply:
Just fixed the XLIBS and XDEFINES paths